app/controllers/staff/level_b/budgets/uploads_controller.rb

```diff
@@ -1,7 +1,26 @@
 class Staff::LevelB::Budgets::UploadsController < Staff::BaseController
   include Secured
+  include StreamCsvDownload
 
   def new
     authorize :level_b, :budget_upload?
   end
+
+  def show
+    authorize :level_b, :budget_upload?
+
+    headers = [
```

Comment:
Do you think it's too early to encapsulate this logic in a `Budgets::ImportFromCSV` class like we do for other imports? This would give us an object we could interact with like `ImportFromCSV.column_headings`. Were you planning on heading in this direction in the next bit of work?

Reply:
I can't remember the specific thinking for this, but @CristinaRO and I discussed it. I think it was either that we thought it would come if/when needed, or that the budgets headers wouldn't vary (at least for this piece of work/Level B) so it might not need interacting with/modifying elsewhere?

It looks like we need it quite a bit in the `ImportFromCsv` class though, so I guess the question is whether we do that now (if it's not getting ahead of ourselves for this commit) or in the next PR/commit that is doing the work of introducing `ImportFromCsv`?

Reply:
Let's leave this until the next PR that adds more functionality, this shouldn't block things for now.